luminoth/train.py

```diff
@@ -22,9 +22,8 @@
 
 def run(custom_config, model_type, override_params, target='',
         cluster_spec=None, is_chief=True, job_name=None, task_index=None,
-        get_model_fn=get_model, get_dataset_fn=get_dataset):
+        get_dataset_fn=get_dataset, get_model_fn=get_model):
```

Comment:
Any reason why this was changed?

Reply:
no i think I changed it hen merging, I'll reverse it